perfmetrics/scripts/bigquery/bigquery.py

```diff
@@ -0,0 +1,206 @@
+import argparse
```

Comment:
Please add the copyright header here.

Reply:
Can you please tell which header you're talking about? I added a copyright header in requirements.in file. Thanks!